nimbus-shared/src/lib.rs

```diff
@@ -0,0 +1,112 @@
+use schemars::JsonSchema;
+
+pub const VERSION: &'static str = env!("CARGO_PKG_VERSION");
+
+#[derive(JsonSchema)]
+#[allow(non_camel_case_types)]
+pub enum RandomizationUnit {
+    nimbus_id,
+    normandy_id,
+}
+
+#[derive(JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct NimbusExperiment {
```

Comment:
Will this replace the definition of the `Experiment` in `./nimbus/lib.rs`?  If so, my PR (#49) may complicate things.  In order to support a range of versions for the backward-compatibility requirement, I changed `Experiment` to be an enum with variants representing the different schema versions.

It might not be so bad, as I should be able to add a variant that points to this representation, and then we can save the supported "historical" variants as structs named by their version or something.